docs/user_guide/configuration/containerized_server.md

```diff
@@ -76,31 +77,20 @@ You can check that the server was started properly with:
 merlin server status
 ```
 
-The `merlin server start` command will add new files to the local configuration `merlin_server/` folder:
-
-- `merlin_server.pf`: A process file containing information regarding the Redis process
-- `app.yaml`: A new app.yaml file configured specifically for the containerized Redis server that we just started
+The `merlin server start` command will add a new file to the local configuration `merlin_server/` folder: `merlin_server.pf`. This is a process file containing information regarding the Redis process. Additionally, the `merlin server start` command will update the `merlin_server/app.yaml` file that was created when we ran `merlin server init`, so that it has `broker` and `results_backend` sections that point to our started server.
 
 To have Merlin read this server configuration:
 
-=== "Copy Configuration to CWD"
-
-    ```bash
-    cp merlin_server/app.yaml .
-    ```
-
 === "Make This Server Configuration Your Main Configuration"
 
-    If you're going to use the server configuration as your main configuration, it's a good idea to make a backup of your current server configuration (if you have one):
-
     ```bash
-    mv ~/.merlin/app.yaml ~/.merlin/app.yaml.bak
+    merlin config use merlin_server/app.yaml
```

Comment:
nice, do you want to add a note this allows for multiple serve files since it's new?

Reply:
sure I can do that